Date.php

```diff
@@ -4571,7 +4569,7 @@ function dateDiff($po_date, $pb_ignoretime = false)
      * @see      Date_TimeZone::isEquivalent()
      * @since    Method available since Release 1.5.0
      */
-    function inEquivalentTimeZones($po_date1, $po_date2)
+    static function inEquivalentTimeZones($po_date1, $po_date2)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.